packages/ui/src/Field.tsx

```diff
@@ -259,22 +258,23 @@ export const Field = ({
         />
       );
     } else if (type === "number") {
-      return (
-        <TextField
-          disabled={disabled}
-          id={name}
-          placeholderText={placeholder}
-          testID={testID}
-          type="number"
-          value={value}
-          onBlur={(result) => {
-            onBlur && onBlur(result.value);
-          }}
-          onChange={(result) => {
-            onChange(result.value);
-          }}
-        />
-      );
+      throw new Error("Number type is not supported in Field component");
```

Comment:
I must have missed this but why are we removing number?

Reply:
Ah we just hadn't implemented it yet. Added.